test/e2e/specs/site-editor/site-editor-inserter.spec.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * WordPress dependencies
+ */
+const { test } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Block list view', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		await requestUtils.activateTheme( 'emptytheme' );
+	} );
+
+	test.afterAll( async ( { requestUtils } ) => {
+		await requestUtils.activateTheme( 'twentytwentyone' );
+	} );
+
+	test.beforeEach( async ( { admin } ) => {
+		await admin.visitSiteEditor();
+	} );
+
+	test( 'inserter toggle button should toggle global inserter', async ( {
+		page,
+	} ) => {
+		await page
+			.locator( 'role=button[name="Toggle block inserter"i]' )
+			.click();
+		await page
+			.locator( '.edit-site-editor__inserter-panel' )
+			.waitFor( { state: 'visible' } );
```

Comment:
Using class selectors is discouraged. Consider using role selectors instead:

```suggestion
		await expect(
			page.locator( 'role=region[name="Block Library"i]' )
		).toBeVisible();
```

Reply:
Implemented

Reply:
I don't think this is right? I think some code are accidentally removed?

Reply:
Sorry for the mistake its fixed now